test/cfg/posix.c

```diff
@@ -532,7 +537,7 @@ int nullPointer_aio_suspend(const struct aiocb *const aiocb_list[], int nitems,
     return aio_suspend(aiocb_list, nitems, timeout);
 }
 
-#ifdef __linux__
+#ifndef __linux__
```

Comment:
This was inverted. This is only available as a syscall but not as a function. I addressed several other cases of this.